shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1488,6 +1497,24 @@ private void createAndSendWindowChangeEvent(@NonNull SemanticsNode route) {
         sendAccessibilityEvent(event);
     }
 
+    /**
+     * Factory method that creates a {@link AccessibilityEvent#TYPE_WINDOW_CONTENT_CHANGED} and sends
```

Comment:
Doesn't a factory method usually return whatever was made in the factory? This doesn't.

Reply:
Good point. Modified here, and on the method above.